STARTERKIT/primary/tailwind.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  theme: {
+    extend: {},
```

Comment:
here we need to add some extends. according that: https://tailwindcss.com/docs/controlling-file-size/#limiting-your-color-palette

also we need to manage breakpoints somehow